### PR TITLE
[DOCU-1557] Konnect cloud 2.4.1.1 support: fixes and new plugins

### PR DESCRIPTION
**Pull request by lena-larionova**
### Review
@skibum 

### Summary
* Updating language for upgrades
* Updating versions in install topics
* Adding list of plugins and updating compat table + plugin metadata

### Reason
Some missed content from yesterday.

### Testing
https://deploy-preview-2958--kongdocs.netlify.app/konnect/updates/
https://deploy-preview-2958--kongdocs.netlify.app/konnect-platform/compatibility/plugins/

### CODE REVIEW

File: app/_data/tables/plugin_index.yml [hunk 1]
```diff
@@ -469,8 +463,8 @@
       free: No
       plus: No
```
Comment (skibum) [inline app/_data/tables/plugin_index.yml:464]: Based on this doc https://docs.google.com/spreadsheets/d/1DLtOGCFL_eQAUDhvucYaJ6LBFwSSm3LMEgg6032-LNM/edit#gid=1122796130 we set Response Transformer to available in free, plus and ent. Looks like this doesn't match the pricing sheet

Comment (skibum) [reply app/_data/tables/plugin_index.yml:464]: Unless I'm misreading the way this content is formatted.

**Review by skibum — APPROVED** [review]
Made a comment on one of the plugin and its tiers but otherwise looks good.

File: app/_data/tables/plugin_index.yml (hunk 1 above)
Comment (lena-larionova) [reply app/_data/tables/plugin_index.yml:464]: I think you might be looking at Response Transformer Advanced. It's harder to tell with the way github shows the preview, but here's the full file+those lines: https://github.com/Kong/docs.konghq.com/blob/d1a4f095aab3648c27d1df36faef4c6a9aa03a14/app/_data/tables/plugin_index.yml#L462

Comment (skibum) [reply app/_data/tables/plugin_index.yml:464]: Great, I thought that may be the case but just wanted to confirm.

File: app/konnect/updates.md [hunk 2]
```diff
@@ -13,12 +13,30 @@ services.
 ### 2021.06.21
 **{{site.base_gateway}} 2.4.1.1 support**
 : {{site.konnect_saas}} now supports {{site.base_gateway}} 2.4.1.1
-runtimes. There is no upgrade path for existing runtimes.
+runtimes. You can keep using existing 2.3.x runtimes, or you can upgrade to
+2.4.1.1 to take advantage of any new features.
 : The 2.4.1.1 release includes two new plugins: [OPA](/hub/kong-inc/opa) and
 [Mocking](/hub/kong-inc/mocking). To use these plugins, and any other features
-newly introduced in this release, [reprovision a new runtime](/konnect/runtime-manager/upgrade).
+newly introduced in this release, [simply start up a new runtime](/konnect/runtime-manager/upgrade).
```
Comment (HCloward) [inline app/konnect/updates.md:20]:
```suggestion
newly introduced in this release, [start up a new runtime](/konnect/runtime-manager/upgrade).
```

**Review by HCloward — APPROVED** [review]
One minor suggestion about removing "simply" - I have always been told to not use it as it can be read as condescending, but not sure what the team's stance is on it. Otherwise, all links work and everything looks good.

**Pull request merged**